web-ui/src/main/resources/catalog/views/default/templates/recordView.html

```diff
@@ -322,7 +322,10 @@ <h2 data-translate="">technicalInformation</h2>
             <th data-translate="">crs</th>
             <td>
               <ul>
-                <li data-ng-repeat="r in ::mdView.current.record.crs">{{r}}</li>
+                <li data-ng-repeat="r in ::mdView.current.record.crs">
+                  <span ng-if="r.name != ''">{{ r.name }} ({{ r.code }})</span>
```

Comment:
After testing it again I can guarantee you that doing `record[field]` = angular.fromJson(record[field])` is not necessary. I'll still add this field to `CatalogService.js` for the sake of readability though.